### PR TITLE
node-db-migration V0.9.x ssl

### PR DESCRIPTION
**Pull request by jamesongithub**

### CODE REVIEW

**Comment by jamesongithub** [comment]
what is the significance of 0.9.x

File: package.json [hunk 1]
```diff
@@ -28,8 +28,13 @@
     "dotenv": "~0.5.1",
     "final-fs": "^1.6.0",
     "mkdirp": "~0.5.0",
+<<<<<<< HEAD
+    "moment": "~2.9.0",
+    "mongodb": "^1.4.35",
+=======
     "moment": "^2.15.1",
     "mongodb": "^1.4.30",
+>>>>>>> origin/v0.9.x
```
Comment (eugeneiiim) [inline package.json]: fix merge conflicts

File: lib/driver/mongodb.js [hunk 2]
```diff
@@ -383,6 +387,44 @@ exports.connect = function(config, callback) {
...
+    sslValidate: true,
+    sslCA: [fs.readFileSync(config.sslCA)]
+  };
+
+  var servers = [];
+  if(config.replSet !== undefined) {
+    var serverNames = config.replSet.split(',');
+
+    for(var i = 0; i < serverNames.length; ++i) {
+      servers.push(new Server(serverNames[i], port, options));
```
Comment (eugeneiiim) [inline lib/driver/mongodb.js]: serverNames.map(function(serveName) { ... })

**Review by eugeneiiim — CHANGES_REQUESTED** [review]

File: lib/driver/mongodb.js [hunk 3]
```diff
@@ -383,6 +387,42 @@ exports.connect = function(config, callback) {
...
+  };
+
+  var servers = [];
+  if(config.replSet !== undefined) {
+    servers = config.replSet.split(',').map(function(member) {
+      return new Server(member, port, options);
+    });
+  } else {
+    servers.push(new Server(host, port, options));
+  }
```
Comment (eugeneiiim) [inline lib/driver/mongodb.js]:
```
var hosts = config.replSet !== undefined ? config.replSet.split(',') : [host];
var servers = hosts.map(function(host) {
  return new Server(host, port, options);
});
```

**Comment by eugeneiiim** [comment]
lgtm contingent

**Review by eugeneiiim — APPROVED** [review]

**Pull request merged**